src/dbus_api.rs

```diff
@@ -53,9 +53,9 @@ impl DbusContext {
             engine: engine.clone(),
         }
     }
-    pub fn get_next_id(&mut self) -> String {
+    pub fn get_next_id(&mut self) -> u64 {
         self.next_index += 1;
-        self.next_index.to_string()
+        self.next_index
```

Comment:
Looks good. BTW My immediate reaction was "why isn't the callsite in this commit too, surely we need to change that as well?" but we don't because it already had a (previously extraneous) `to_string()` call.